doc/source/reference/random/examples/cffi.rst

```diff
@@ -1,5 +1,7 @@
 Extending via CFFI
 ------------------
 
+The source of ``numpy/random/_examples/cffi/extending.py``
```

Comment:
I don't think this should be added. The whole point of using an underscore is that it's private. Users should be getting this from the docs, not from wherever on their system NumPy is installed. 

I suggest removing the 3 instances in this PR, as well as the instance of `random.examples` that's still left on https://10421-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/reference/random/extending.html

Reply:
I added this to clarify that we had not removed the examples as had been [elsewhere mentioned](numba). Removing as requested.

Reply:
still one left, will fix later. first getting this in before the 1.18.x branch